functions/Set-DbaDbFileSize.ps1

```diff
@@ -0,0 +1,121 @@
+function Set-DbaDbFileSize {
+    <#
+    .SYNOPSIS
+        Sets databases to a non-default growth and growth type. 64MB by default.
+
+    .DESCRIPTION
+        Sets databases to a non-default growth and growth type. 64MB by default.
+
+    .PARAMETER SqlInstance
+        The target SQL Server instance or instances.
+
+    .PARAMETER SqlCredential
+        Login to the target instance using alternative credentials. Accepts PowerShell credentials (Get-Credential).
+
+        Windows Authentication, SQL Server Authentication, Active Directory - Password, and Active Directory - Integrated are all supported.
+
+        For MFA support, please use Connect-DbaInstance.
+
+    .PARAMETER Database
+        The name of the target databases
+
+    .PARAMETER GrowthType
+        The growth type, MB by default - valid values are MB, KB, GB or TB. MB by default
```

Comment:
What about percent?

Reply:
We follow best practices, and percent should not be used 😉

Reply:
Oh, yes. I agree.